docs/mddocs/Quickstart/open_webui_with_ollama_quickstart.md

```diff
@@ -163,10 +163,16 @@ Go to **Settings -> Models** in the menu, choose a model under **Pull a model fr
   <img src="https://llm-assets.readthedocs.io/en/latest/_images/open_webui_pull_models.png" width="100%" />
 </a>
 
+> [!NOTE]
+> On latest UI (v0.4.8), the relative settings have been moved to **Admin Settings -> Connections -> Manage Ollama API Connections -> Manage**.
```

Comment:
Change to `On the latest version of Open WebUI`.

Reply:
done.